src/ThemeProvider.tsx

```diff
@@ -154,7 +154,10 @@ function applyColorScheme(theme: Theme, colorScheme: string) {
   if (!theme.colorSchemes[colorScheme]) {
     // eslint-disable-next-line no-console
     console.error(`\`${colorScheme}\` scheme not defined in \`theme.colorSchemes\``)
-    return theme
+
+    // Apply the first defined color scheme
+    const defaultColorScheme = Object.keys(theme.colorSchemes)[0]
```

Comment:
I wonder if index `0` is [magic-numbery](https://en.wikipedia.org/wiki/Magic_number_(programming))? Though it may be our best option, since [the `Theme` type definition](https://github.com/primer/components/blob/f0f8edc7f55ac243d58c76e8346072886299cedb/src/ThemeProvider.tsx#L11) doesn’t require a default scheme to be designated (example below). 🤔 

```ts
export type Theme = {
  colorSchemes: {
    default: { [key: string]: any }
    [key: string]: any
  }
  [key: string]: any
}
```

Reply:
I agree that `[0]` feels a little bit wrong here. Unfortunetly, we can't make any assumptions about the format of the `theme` object without making a breaking change to the library. Let's move forward with this "incrementally correct" solution, and we can revisit TypeScript types for the `theme` object later.